docs/product_release.md

```diff
@@ -0,0 +1,44 @@
+# Product Release
+
+Tractus-X products do follow their own release process. Please check the leading product repositories for details.
+Independent of the technical details of product release processes, the released artifacts and minimum level of quality ensurance
+is the same across all of them.
+
+The release contents for individual products are are planned in the [open refinement and planning](./open-refinement-and-planning.md) sessions.
+These sessions define the scope for all dataspace components, but therefore paint a clear picture of enhancements necessary for every single product.
+The product teams themselfes are then responsible to define fine-grained task breakdowns as suiting their own needs.
+
+> __HINT:__
+> If product enhancements contribute to an overarching dataspace feature, please link the dataspace feature defined in `sig-release` in your product issue.
+> These links can be done in either the description or any comment on your issue. Use `eclipse-tractusx/sig-release#issue-number`
+
+Once planning activities for a certain iteration are completed, the implementation phase is starting and product teams can coordinate, where necessary
+through our [communication channels](https://eclipse-tractusx.github.io/docs/oss/getting-started#communication-channels).
+
+Integration tests are performed by maintaining an umbrella Chart, that deploys all relevant dataspace components to spin up an artificial dataspace based on Tractus-X components.
+There is a dedicated repository [eclipse-tractusx/tractus-x-umbrella](https://github.com/eclipse-tractusx/tractus-x-umbrella), that is maintained collectively to reflect the latest product release versions.
+
+Additional to the integration- and unit-test suites the teams maintain, every product also needs to adhere to the [Tractus-X release guidelines (TRGs)](https://eclipse-tractusx.github.io/docs/release).
```

Comment:
Proposal for line 21: Every product needs to adhere...
Proposal to insert between 24 & 25: this is in addition to...